package.json

```diff
@@ -22,7 +22,7 @@
     "cross-env": "^3.0.0",
     "css-loader": "^0.25.0",
     "file-loader": "^0.9.0",
-    "node-sass": "^4.5.0",
+    "node-sass": "^4.11.0",
```

Comment:
@non didn't you say node-sass was missing altogether? or was it just out of date? or something else? just curious